leasing/report/invoice/invoicing_review.py

```diff
@@ -78,159 +78,166 @@ class Labels:
     """,
     "rent_info_not_complete": """
         SELECT NULL AS "section",
-               li.identifier AS "lease_id",
-               l.start_date,
-               l.end_date
-          FROM leasing_lease l
-               INNER JOIN leasing_leaseidentifier li
-               ON l.identifier_id = li.id
-         WHERE (l.end_date IS NULL OR l.end_date >= %(today)s)
-           AND l.start_date IS NOT NULL
-           AND l.service_unit_id = ANY(%(service_units)s)
-           AND l.deleted IS NULL
-           AND l.state IN ('lease', 'short_term_lease', 'long_term_lease')
-           AND l.is_rent_info_complete = FALSE
-         GROUP BY l.id,
-                 li.id
-         ORDER BY li.identifier;
+            li.identifier AS "lease_id",
+            l.start_date,
+            l.end_date
+        FROM leasing_lease l
+        INNER JOIN leasing_leaseidentifier li
+            ON l.identifier_id = li.id
+        WHERE (l.end_date IS NULL OR l.end_date >= %(today)s)
+            AND l.start_date IS NOT NULL
+            AND l.service_unit_id = ANY(%(service_units)s)
+            AND l.deleted IS NULL
+            AND l.state IN ('lease', 'short_term_lease', 'long_term_lease')
+            AND l.is_rent_info_complete = FALSE
+        GROUP BY l.id,
+                li.id
+        ORDER BY li.identifier;
     """,
     "no_rents": """
         SELECT NULL AS "section",
-               li.identifier AS "lease_id",
-               l.start_date,
-               l.end_date
+            li.identifier AS "lease_id",
+            l.start_date,
+            l.end_date
         FROM leasing_lease l
-               LEFT OUTER JOIN leasing_rent r
-               ON l.id = r.lease_id
-                  AND r.deleted IS NULL
-                  AND (r.start_date IS NULL OR r.start_date <= %(today)s)
-                  AND (r.end_date IS NULL OR r.end_date >= %(today)s)
-                  INNER JOIN leasing_leaseidentifier li
-                  ON (l.identifier_id = li.id)
+        LEFT OUTER JOIN leasing_rent r
+            ON l.id = r.lease_id
+                AND r.deleted IS NULL
+                AND (r.start_date IS NULL OR r.start_date <= %(today)s)
+                AND (r.end_date IS NULL OR r.end_date >= %(today)s)
+        INNER JOIN leasing_leaseidentifier li
+            ON (l.identifier_id = li.id)
         WHERE (l.end_date IS NULL OR l.end_date >= %(today)s)
-          AND l.service_unit_id = ANY(%(service_units)s)
-          AND l.deleted IS NULL
+            AND l.service_unit_id = ANY(%(service_units)s)
+            AND l.deleted IS NULL
+            AND l.state NOT IN ('reservation', 'power_of_attorney')
```

Comment:
This seems to repeat multiple times, it could be considered for the value to be bassed to `cursor.execute` similarly to `today` and `service_unit`
Not 100% whether those can be left unused if they are not in the SQL, I suppose it should work though.

This seems to do what was requested, so good job!